priv/repo/migrations/20160810124357_create_project.exs

```diff
@@ -0,0 +1,19 @@
+defmodule CodeCorps.Repo.Migrations.CreateProject do
+  use Ecto.Migration
+
+  def change do
+  	create table(:projects) do
+      add :description, :string
+      add :icon_large_url, :string
+      add :icon_thumb_url, :string
+      add :long_description_body, :string
+      add :long_description_markdown, :string
+      add :slug, :string
+      add :title, :string
```

Comment:
Agreed on both parts. I don't think anything else needs to be non-nullable, though.